### PR TITLE
Add possibility to additional configuration of the main context

### PR DESCRIPTION
**Pull request by realmyst**
Now we can add more options to main config context, e.g.:
```
user www-data  www-data;
worker_processes  auto;
pid /var/run/nginx.pid;
worker_rlimit_nofile 1024;
include /etc/nginx/modules-enabled/*.conf; # extra option

http {
...
}
```

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -34,6 +34,7 @@ nginx_worker_rlimit_nofile: 1024
 nginx_log_dir: "/var/log/nginx"
 nginx_error_log_level: "error"
 
+nginx_extra_params: ""
```
Comment (jdauphant) [inline defaults/main.yml]: nginx_extra_params
Maybe we can make this more explicit like "nginx_extra_root_params" or "nginx_extra_top_params"

Comment (realmyst) [inline defaults/main.yml]: Yes, I agree that `nginx_extra_root_params` would be more explicit. Renamed it.

**Comment by jdauphant** [comment]
Merged and release in 2.4

**Comment by jdauphant** [comment]
@realmyst I didn't see it before merging but could have been nice if it was a list instead of a single string attributes

**Pull request merged**